### ISSUE TITLE
SCTE links broken

### ISSUE THREAD

**Issue by chrisn** [tobie/specref#542]
Links to SCTE standards aren't working. For example: [SCTE35](https://www.specref.org/?q=scte35) returns:

`https://www.scte.org/SCTEDocs/Standards/SCTE%252035%25202019r1.pdf`

which returns a 400 error. The following URL works:

`https://www.scte.org/SCTEDocs/Standards/SCTE%2035%202019r1.pdf`

The same issue affects other SCTE documents I'm referencing.

Not sure whether the problem is in specref itself or the upstream server that's providing the SCTE references (cc @sandersaares).

**Comment by tobie** [comment]
Oh, is there double escaping going on somewhere? That's bad.

**Comment by chrisn** [comment]
The links coming from the [upstream server](https://dashifspecref.blob.core.windows.net/scte/scte.json?st=2019-04-01T00%3A00%3A00Z&se=2039-04-01T00%3A00%3A00Z&sp=rl&sv=2018-03-28&sr=b&sig=eIrXcpSm6WkcgRNBgNKlyMnIUGWLwgTurzYy8K2WApI%3D) look OK.

**Comment by tobie** [comment]
I'm a bit confused because I thought we had deliberately fixed this in https://github.com/tobie/specref/issues/528 and https://github.com/tobie/specref/pull/529. So I guess the answer is to back-down from this fix?

**Comment by sandersaares** [comment]
Ah yes looks like some wires got crossed in the original continuity. Here is what I observed happening originally:

1. Without URL-escaping, the URLs were rejected by `npm test`
1. So I added URL-escaping to the URLs in the upstream file.
1. But then fetch-refs removed the URL-escaping.
1. Removing the URL-escaping by fetch-refs will make `npm test` fail again.

I was under the impression the escaping-related commit was intended to fix point 3 but perhaps I did not sufficiently understand the change.

Whatever the case, with the present codebase I am no longer able to reproduce point 3 on my PC - URL-escaping is not getting removed anymore. I guess something in the logic changed meanwhile? Or perhaps my original observation was in error. Well, seems to be OK now at least!
